src/mechanics/timeIntegration/RungeKuttaBase.h

```diff
@@ -38,7 +38,22 @@ class RungeKuttaBase : public TimeIntegrationBase
     //! @brief ... return, if time (e.g. for the calculation of external loads) has changed
     virtual bool HasTimeChanged(int rStage) const = 0;
 
+
+    // temporary fix
+    //! @brief sets the  time step for the time integration procedure (initial value)
+    virtual void SetTimeStep(double rTimeStep) override
```

Comment:
Because all time related stuff is now done by the TimeControl class. But it is not that simple to replace them in all derived time integration schemes. Plan is to that step by step.